pandas/core/generic.py

```diff
@@ -8875,13 +8875,24 @@ def _add_series_only_operations(cls):
         def nanptp(values, axis=0, skipna=True):
             nmax = nanops.nanmax(values, axis, skipna)
             nmin = nanops.nanmin(values, axis, skipna)
+            warnings.warn("Method .ptp is deprecated and will be removed "
+                          "in a future version. Use numpy.ptp instead.",
+                          FutureWarning, stacklevel=4)
             return nmax - nmin
 
         cls.ptp = _make_stat_function(
             cls, 'ptp', name, name2, axis_descr,
-            """Returns the difference between the maximum value and the
+            """
+            .. deprecated:: 0.24.0
+                Use numpy.ptp instead
+
+            Returns the difference between the maximum value and the
             minimum value in the object. This is the equivalent of the
-            ``numpy.ndarray`` method ``ptp``.""",
+            ``numpy.ndarray`` method ``ptp``.
+
+            .. deprecated:: 0.24.0
```

Comment:
hmm this is duplicated here

Reply:
This is based on comment from @jorisvandenbossche 
`This needs a blank line above this one. Can you also put the deprecated part below the text?` - waiting for his reply if the edit was as expected

Reply:
My bad - misinterpreted his review and duplicated in hurry - fixing now

Reply:
Done thanks